Week2/homework/JS-Exercises/Exercise-5-Who-wants-a-drink.js

```diff
@@ -0,0 +1,14 @@
+let drinkTray = [];
+
+const drinkTypes = ['cola', 'lemonade', 'water'];
+
+for (i = 0 ; i < 5 ; i++) {
+    if (i <3) {
+        drinkTray.push(drinkTypes[i]);
+    }
+    else if (i >= 3) {
+        drinkTray.push(drinkTypes[i % 3]);
+    }
+};
```

Comment:
Ah. This works fine but it also could be further reduced in complexity to something like: `drinkTray.push(drinkTypes[i % drinkTypes.length]);`

Then we don't need the if-else blocks and the code works the same way as it is doing now 🙂